encord/http/v2/api_client.py

```diff
@@ -123,30 +127,40 @@ def patch(
     ) -> T:
         return self._request_with_payload("PATCH", path, params, payload, result_type)
 
+    def _serialise_payload(self, payload: Union[BaseDTO, Sequence[BaseDTO], None]) -> Optional[dict]:
+        if isinstance(payload, list):
+            return [p.to_dict() for p in payload]  # type: ignore
```

Comment:
Typing is wrong here? Returning union to_dict doesn't work?

Reply:
Yes, apparently had a brain glitch, fixed! :)